src/Symfony/Component/Console/Application.php

```diff
@@ -934,7 +935,7 @@ protected function configureIO(InputInterface $input, OutputInterface $output):
                 break;
         }
 
-        if (true === $input->hasParameterOption(['--quiet', '-q'], true)) {
+        if (true === $input->hasParameterOption(['--quiet', '-q'], true) || $this->isDirectOutputDisabled()) {
```

Comment:
the same code path can be triggered using SHELL_VERBOSITY=-1 isnt it?

https://github.com/symfony/symfony/blob/7.1/src/Symfony/Component/Console/Application.php#L919-L921

Reply:
i'd like to avoid SYMFONY_CONSOLE_OUTPUT vs SHELL_VERBOSITY yes :)

Reply:
i understand the BC concern, perhaps we need to reconsider what --quiet really means :sweat_smile:

Reply:
I agree that expanding `SHELL_VERBOSITY` modes would be better than a new env var.

> SHELL_VERBOSITY=-2 maybe?
👍 and `--silent` maybe

Reply:
i'd consider making -q really quiet

i see no reason to preserve just error output with -q and proper error logging setup

Reply:
I've submitted the alternative in a new PR (to not mix up the discussions): #53632